codemeta/codemeta.py

```diff
@@ -301,16 +301,17 @@ def build(**kwargs):
             #source = source.replace("https://github.com/","")
             #source = source.replace("git@github.com:","")
             #e.g. transform git@gitlab.com/X in https://gitlab.com/X
+            repo_kind=("")
```

Comment:
This is weird syntax, why the ``()``, it has no function here?  I see you've done it in more places.

Reply:
Ah, I think you meant for it to be a tuple.... judging from your type annotation.. but it isn't (and I don't think it should be)

If you'd want it to be a tuple the syntax would have been:

```python
repo_kind=("",)
```

(the trailing comma makes the difference, it's a bit of a python quirck)

Reply:
yes it'a tuple, to get the modified value back from the function respect to a string.
I will add the comma